lib/chat/getConversations.js

```diff
@@ -37,6 +43,20 @@ exports.func = (args) => {
         return entry
       })
 
+      const nonexistentConversations = []
+
+      for (const conversation in conversationIds) {
+        if (!response.find((res) => res.id === conversation)) {
+          nonexistentConversations.push(conversation)
+        }
+      }
+
+      if (response.length < conversationIds.length) {
```

Comment:
Is it better to do this error handling for the user, or to return the data with a caveat that non-existent conversations won't be in the response?
This strikes me as throwing away data unnecessarily but open to discussion

Reply:
Honestly we could do either but it already throws if one of them exists but can't be accessed, which would mean inconsistent behavior.

Yes we are throwing away the ones that do get returned, but this behavior is undocumented and is most likely a nasty surprise to anyone that comes across it. I also struggle to think of a situation where someone would regularly hit this.

People also tend to not pay much attention to the docs so maybe we could place a warning there but I'm not sure how useful it would be.